nixos/release.nix

```diff
@@ -360,6 +360,7 @@ in rec {
   tests.netdata = callTest tests/netdata.nix { };
   tests.networking.networkd = callSubTests tests/networking.nix { networkd = true; };
   tests.networking.scripted = callSubTests tests/networking.nix { networkd = false; };
+  tests.nextcloud = callTest tests/nextcloud.nix { };
```

Comment:
should that be `callSubTests` now?

Reply:
Maybe.  Either way this path needs to be adjusted :sweat_smile: